foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-docker/pom.xml

```diff
@@ -63,19 +63,14 @@
             <plugin>
                 <groupId>io.fabric8</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>default-build</id>
-                        <configuration>
-                            <skip>true</skip>
-                            <images>
-                                <image>
-                                    <name>${project.artifactId}:${project.version}</name>
-                                </image>
-                            </images>
-                        </configuration>
-                    </execution>
-                </executions>
+                <configuration>
+                    <skip>true</skip>
+                    <images>
+                        <image>
+                            <name>${project.artifactId}:${project.version}</name>
+                        </image>
+                    </images>
+                </configuration>
```

Comment:
Q: Why set the image name when the config is set to skip?

Reply:
This was brought up at DOD and we as a team need to decide if we want to unify this configuration. Will be covered in different ticket